src/views/blocks/rsvp/details/availability.php

```diff
@@ -8,23 +8,32 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link  {INSERT_ARTICLE_LINK_HERE}
  *
  * @since 4.9.3
- * @version 4.9.4
+ * @since TBD Corrected amount of available/remaining tickets.
  *
+ * @version TBD
  */
 
-$remaining_tickets = $ticket->remaining();
-$is_unlimited = -1 === $remaining_tickets;
+/** @var Tribe__Tickets__Ticket_Object $ticket */
+if ( empty( $ticket->ID ) ) {
+	return;
+}
+
+/** @var Tribe__Tickets__Tickets_Handler $tickets_handler */
+$tickets_handler = tribe( 'tickets.handler' );
+
+$available = $tickets_handler->get_ticket_max_purchase( $ticket->ID );
+
 ?>
 <div class="tribe-block__rsvp__availability">
 	<?php if ( ! $ticket->is_in_stock() ) : ?>
 		<span class="tribe-block__rsvp__no-stock"><?php esc_html_e( 'Out of stock!', 'event-tickets' ); ?></span>
-	<?php elseif ( ! $is_unlimited ) : ?>
-		<span class="tribe-block__rsvp__quantity"><?php echo $ticket->remaining(); ?> </span>
+	<?php elseif ( -1 !== $available ) : ?>
+		<span class="tribe-block__rsvp__quantity"><?php echo $available; ?> </span>
```

Comment:
Yes we know this is probably an integer, but we should still late escape (not your bug initially)

```suggestion
		<span class="tribe-block__rsvp__quantity"><?php echo esc_html( $available ); ?> </span>
```